jinahub/indexers/storage/MongoDBStorage/tests/test_mongodb.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+import pytest
+import numpy as np
+from jina import Document, DocumentArray, Flow
+from jina_commons.indexers.dump import import_vectors, import_metas
+
+from .. import doc_without_embedding
+
+NUM_DOCS = 10
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+compose_yml = os.path.abspath(os.path.join(cur_dir, 'docker-compose.yml'))
+
+
+@pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
```

Comment:
Can you add a test for loading from config? For example https://github.com/jina-ai/executors/blob/4c202a563a67b9d07181911488a36b1e23a54f99/jinahub/encoders/text/DPRTextEncoder/tests/unit/test_encoder.py#L25-L30

but use `Path(__file__).parents[1] / 'config.yml'` to be precise about the location of config file